GoMeet/src/main/webapp/test/spec/specCreateMeetingShared.js

```diff
@@ -0,0 +1,190 @@
+class MockHTMLCollection {
+  /**
+   * Construct a MockHTMLCollection object, a fake that mocks
```

Comment:
nit: fakes and mocks are often different things... https://stackoverflow.com/a/55030455